BuildScripts/CopyProjDirectory.bat

```diff
@@ -0,0 +1,6 @@
+rem File "d:\checkouts\D-FAST\D-FAST_Bank_Erosion\dfastbe.dist\pyproj\datadir.py", line 109, in get_data_dir
```

Comment:
willen we dit er zo bijzetten?

miss handig om aan te geven dat nuitka deze files in zijn build nodig heeft en naar verwijst?